ambry-replication/src/main/java/com/github/ambry/replication/ReplicationManager.java

```diff
@@ -278,15 +280,20 @@ public void onPartitionBecomeStandbyFromBootstrap(String partitionName) {
         throw new StateTransitionException(
             "Store " + partitionName + " is not started during Bootstrap-To-Standby transition", StoreNotStarted);
       }
+      logger.info("Partition = {}, replicaId = {}, storeId = {}", partitionName, localReplica, store);
       // 2. check if store is new added and needs to catch up with peer replicas.
+      ReplicaState currentState = store.getCurrentState();
```

Comment:
nit: This variable is used only once in either if() or else block.  There's no benefit for introducing a variable.

Reply:
Its used in one more place L296